lib/spark_api/models/email_link.rb

```diff
@@ -0,0 +1,21 @@
+module SparkApi
+  module Models
+    class EmailLink < Base
+
+      extend Finders
+
+      self.prefix = "/flexmls/"
+      self.element_name = "emaillinks"
+
+      attr_accessor :template, :sort_id
+
+      def filter
+        "EmailLink Eq '#{id}'"
+      end
```

Comment:
A `Filter` attribute was added to EmailLinks since this was written - http://sparkplatform.com/docs/api_services/flexmls/email_links#response It may not actually effect the listing results, but it seems a little weird that `filter` would return "EmailLink Eq '#{id}'" and `Filter` would return a normal filter string with the search criteria.